ext/node/polyfills/worker_threads.ts

```diff
@@ -137,10 +136,14 @@ class _Worker extends EventEmitter {
         specifier = toFileUrl(specifier as string);
       }
     }
+
+    // TODO(bartlomieu): this doesn't match the Node.js behavior, it should be
+    // `[worker {threadId}] {name}` or empty string.
+    const name = (options?.name ?? "").trim();
```

Comment:
I think if `options.eval` is set, this needs to be `[worker eval]`

Reply:
Thanks, applied